src/components/ConfirmationPopup.js

```diff
@@ -0,0 +1,25 @@
+import Popup from "./Popup";
+
+class ConfirmationPopup extends Popup {
+  constructor(popupSelector) {
+    super({ popupSelector });
+  }
+  setSubmitAction(submitHandler){
+    this.submitHandler = submitHandler;
+
+  }
+  setDefaultSaveButton(){
+    this._saveButton.textContent = "Yes";
+  }
+  updateSaveButton(){
+    this._saveButton.textContent = "Deleting...";
+  }
+
+  setEventListeners() {
+    super.setEventListeners()
+    this._saveButton = this._popupElement.querySelector(".modal__save-button");
+      this._saveButton.addEventListener("click", () => {this.submitHandler(), this._saveButton.textContent = "Saving..."});
```

Comment:
`this._saveButton.textContent = "Saving..."}` is not needed here


You have `setDefaultSaveButton` and `updateSaveButton` for that